streams/buffer.ts

```diff
@@ -315,7 +315,9 @@ export class Buffer {
       return;
     }
     if (n < 0 || n > this.length) {
-      throw Error("bytes.Buffer: truncation out of range");
+      throw Error(
+        `Error truncating buffer, truncation value "${n}" out of range: value must be between 0 and ${this.length}`,
```

Comment:
```suggestion
        `Cannot truncate the buffer as the truncation value "${n}" is out of range: value must be between 0 and ${this.length}`,
```

Reply:
Seems a little long, no?

Reply:
But that is what the style guide now recommend

Reply:
How about `Buffer truncation value "${n}" is out of range: value must be between 0 and ${this.length}`

Reply:
I applied this